app/Http/Controllers/Api/V1/AuthController.php

```diff
@@ -114,60 +116,100 @@ public function signUp(SignUpRequest $request): JsonResponse
         return isset($data['errors']) ? $this->error($data) : $this->success($data, 200);
     }
 
+    #[OA\Get(
+        path: '/api/v1/email/verify',
+        operationId: 'authVerifyEmail',
+        tags: ['Auth'],
+        parameters: [
+            new OA\Parameter(
+                name: 'X-Requested-With',
+                in: 'header',
+                required: true,
+                description: 'Custom header for XMLHttpRequest',
+                schema: new OA\Schema(
+                    type: 'string',
+                    default: 'XMLHttpRequest'
+                )
+            ),
+            new OA\Parameter(
+                name: 'id',
+                in: 'query',
+            ),
+            new OA\Parameter(
+                name: 'hash',
+                in: 'query',
+            ),
+            new OA\Parameter(
+                name: 'signature',
+                in: 'query',
+            ),
+            new OA\Parameter(
+                name: 'expires',
+                in: 'query',
+            ),
+        ],
+        responses: [
+            new OA\Response(response: '200', description: 'Success.'),
+            new OA\Response(response: '400', description: 'Validation errors!'),
+        ],
+    )]
+    public function verifyEmail(Request $request)
+    {
+        $data = $this->authService->verifyEmail($request);
```

Comment:
@damini-7span $request should the $request->all()

Reply:
As we discussed, We have to take object rather than array because array is giving error in laravel's built in function of verify email.